frontend/__tests__/stores/cloudProfile.spec.js

```diff
@@ -73,472 +78,442 @@ describe('stores', () => {
+      }
+      const supported17VersionWithExpirationWarning = {
+        version: '1.17.0',
+        classification: 'supported',
+        expirationDate: '2024-01-15T23:59:59Z', // not expired but expiration warning
+      }
+      const preview166Version = {
+        version: '1.16.6',
+        classification: 'preview',
+      }
+      const supported165VersionWithExpiration = {
+        expirationDate: '2024-04-12T23:59:59Z', // not expired
+        version: '1.16.5',
+        classification: 'supported',
+      }
+      const unclassified164VersionWithExpiration = {
+        expirationDate: '2024-04-12T23:59:59Z', // not expired
+        version: '1.16.4',
+      }
+      const deprecatedVersion = {
+        version: '1.16.3',
+        classification: 'deprecated',
+      }
+      const supported162VersionWithExpirationWarning = {
+        expirationDate: '2024-01-15T23:59:59Z', // not expired but expiration warning
+        version: '1.16.2',
+        classification: 'supported',
+      }
+      const expiredVersion = {
+        expirationDate: '2023-03-15T23:59:59Z', // expired
+        version: '1.16.1',
+      }
+      const deprecatedOldestVersion = {
+        version: '1.16.0',
+        classification: 'deprecated',
+      }
+      const invalidVersion = {
+        version: '1.06.2',
+      }
+      const kubernetesVersions = [
+        supported165VersionWithExpiration, // 1.16.5 on top to test sorting
+        preview22Version, // 2.1.0
+        supported20Version, // 2.0.0
+        supported18VersionWithExpirationWarning, // 1.18.0
+        deprecated181VersionWithnoExpiration, // 1.18.1
+        supported17VersionWithExpirationWarning, // 1.17.0
+        preview166Version, // 1.16.6
+        unclassified164VersionWithExpiration, // 1.16.4
+        deprecatedVersion, // 1.16.3
+        supported162VersionWithExpirationWarning, // 1.16.2
+        expiredVersion, // 1.16.1
+        invalidVersion, // 1.06.2 not semver compatible
+        deprecatedOldestVersion, // 1.16.0
+      ]
+
+      beforeEach(() => {
+        setKubernetesVersions(kubernetesVersions)
+      })
+
       describe('#sortedKubernetesVersions', () => {
-        const kubernetesVersions = [
-          {
-            version: '1.13.4',
-            classification: 'deprecated',
-          },
-          {
-            version: '1.14.0',
-          },
-          {
-            expirationDate: '2120-04-12T23:59:59Z', // not expired
-            version: '1.16.3',
-            classification: 'supported',
-          },
-          {
-            expirationDate: '2019-03-15T23:59:59Z', // expired
-            version: '1.16.2',
-          },
-          {
-            version: '1.06.2', // invalid version (not semver compatible)
-          },
-        ]
+        it('should filter and sort kubernetes versions from cloud profile', () => {
+          const decoratedAndSortedVersions = cloudProfileStore.sortedKubernetesVersions('foo')
```

Comment:
rename to `sortedVersions`/`sortedKubernetesVersions`?

Reply:
see comment above